lib/documents/schemas/utaac_decisions.json

```diff
@@ -2034,6 +2034,42 @@
           "label": "Tribunal procedure and practice - tribunal practice",
           "value": "tribunal-procedure-and-practice-tribunal-practice"
         },
+        {
+          "label": "Universal Credit - Sanctions",
+          "value": "universal_credit-sanctions"
```

Comment:
In the other `value`s, we seem to use dashes to separate words even if the whole label has got a dash in it (and not use underscores and dashes). Although I think this is clearer, is it a problem that it's not consistent with the others?

Reply:
Ah good point, I'll go with consistency :+1: